src/foam/u2/view/ArrayView.js

```diff
@@ -119,6 +141,13 @@ foam.CLASS({
   `,
 
   methods: [
+    function init() {
+      if ( ! this.allowDuplicates) {
+        // Hook up disabled data for any array that does not allow duplicates
+        this.disabledData_$.linkFrom(this.data$);
+      }
+    },
```

Comment:
can disabledData_ be an expression conditional on allowDuplicates so it doesn't need this hook up?